include/roaring/bitset/bitset.h

```diff
@@ -216,65 +216,13 @@ inline bool bitset_next_set_bit(const bitset_t *bitset, size_t *i) {
     //.....
   }
   */
-inline size_t bitset_next_set_bits(const bitset_t *bitset, size_t *buffer,
-                                   size_t capacity, size_t *startfrom) {
-    if (capacity == 0) return 0;  // sanity check
-    size_t x = *startfrom / 64;
-    if (x >= bitset->arraysize) {
-        return 0;  // nothing more to iterate over
-    }
-    uint64_t w = bitset->array[x];
-    w >>= (*startfrom & 63);
-    size_t howmany = 0;
-    size_t base = x << 6;
-    while (howmany < capacity) {
-        while (w != 0) {
-            uint64_t t = w & (~w + 1);
-            int r = roaring_trailing_zeroes(w);
-            buffer[howmany++] = r + base;
-            if (howmany == capacity) goto end;
-            w ^= t;
-        }
-        x += 1;
-        if (x == bitset->arraysize) {
-            break;
-        }
-        base += 64;
-        w = bitset->array[x];
-    }
-end:
-    if (howmany > 0) {
-        *startfrom = buffer[howmany - 1];
-    }
-    return howmany;
-}
-
+size_t bitset_next_set_bits(const bitset_t *bitset, size_t *buffer,
+                            size_t capacity, size_t *startfrom);
 typedef bool (*bitset_iterator)(size_t value, void *param);
-
 // return true if uninterrupted
-inline bool bitset_for_each(const bitset_t *b, bitset_iterator iterator,
-                            void *ptr) {
-    size_t base = 0;
-    for (size_t i = 0; i < b->arraysize; ++i) {
-        uint64_t w = b->array[i];
-        while (w != 0) {
-            uint64_t t = w & (~w + 1);
-            int r = roaring_trailing_zeroes(w);
-            if (!iterator(r + base, ptr)) return false;
-            w ^= t;
-        }
-        base += 64;
-    }
-    return true;
-}
+bool bitset_for_each(const bitset_t *b, bitset_iterator iterator,
+                     void *ptr);
 
-inline void bitset_print(const bitset_t *b) {
```

Comment:
This appears to remove the function declaration from the header, we probably want to keep a declaration here, but not provide a definition.